src/widgets/block.rs

```diff
@@ -288,13 +288,26 @@ impl<'a> Block<'a> {
     /// - [`Block::title_alignment`]
     /// - [`Block::title_position`]
     ///
+    /// # Future improvements
+    ///
+    /// In a future release of Ratatui (likely 0.31) this method will be changed to accept
```

Comment:
Minor: imo we don't need to mention `(likely 0.31)`

Reply:
Done in 73918092f